doofinder-for-woocommerce/includes/class-update-on-save.php

```diff
@@ -173,17 +173,17 @@ public static function add_to_update_on_save_db($post_id, $post_type, $action)
 
         $table_name = $wpdb->prefix . 'doofinder_update_on_save';
 
-        $resultado = $wpdb->get_var("SELECT * FROM $table_name WHERE post_id = $post_id");
+        $result = $wpdb->get_var("SELECT * FROM $table_name WHERE post_id = $post_id");
 
-        if ($resultado != null) {
+        if ($result != null) {
             $wpdb->update(
                 $table_name,
                 array(
                     'post_id' => $post_id,
                     'type_post' => $post_type,
                     'type_action' => $action
                 ),
-                array('id' => $resultado)
```

Comment:
But would this change solve this customer, would the ones that were already working well stop working?

Reply:
The fact is that it wasn't working, however, in my case it was throwing a non-fatal error. I attach a line of my debug.log:

`[22-Apr-2024 08:02:00 UTC] WordPress database error Unknown column 'id' in 'where clause' for query UPDATE `wp_doofinder_update_on_save` SET `post_id` = 35, `type_post` = 'product', `type_action` = 'update' WHERE `id` = '35' made by edit_post, wp_update_post, wp_insert_post, do_action('wp_insert_post'), WP_Hook->do_action, WP_Hook->apply_filters, Doofinder\WP\Update_On_Save::Doofinder\WP\{closure}, Doofinder\WP\Update_On_Save::add_item_to_update, Doofinder\WP\Update_On_Save::add_item_to_db, Doofinder\WP\Update_On_Save::add_to_update_on_save_db`